fmriprep/interfaces/utils.py

```diff
@@ -44,7 +44,8 @@ class EstimateReferenceImage(SimpleInterface):
 
     def _run_interface(self, runtime):
         in_nii = nb.load(self.inputs.in_file)
-        global_signal = in_nii.get_data()[:, :, :, :50].mean(axis=0).mean(
+        data_slice = in_nii.dataobj[:, :, :, :50]
+        global_signal = data_slice.mean(axis=0).mean(
             axis=0).mean(axis=0)
```

Comment:
This would break if the file had fewer than 50 timepoints, because reshape needs exact length and indexind does not.